### DISCUSSION TITLE
Updraft Course's Roadmap

### DISCUSSION THREAD

**Discussion by MostPow3rful** [Cyfrin/foundry-full-course-cu#2556] [Q&A]
GM
is there any map to watch updraft courses ?
i am doing as this :
BLockchain basic, solidity 101, foundry 101, foundry advanced, assembly & formal verification, DevOps, Security

is it ok to become a developer(first) and auditor(next) ?

**Comment by EngrPips** [comment]
Hello @MostPow3rful, I haven't seen any explicit map for taking courses on Cyfrin Updraft, but yeah, the order in which you have taken the course is okay, and yeah, you need to be at least comfortable with writing smart contracts before you start bothering yourself about securing them.

**Reply by MostPow3rful** [reply]
Thanks <3

**Reply by EngrPips** [reply]
You are welcome, Friend. Keep crushing it.
